parser/Builder.cc

```diff
@@ -614,10 +614,27 @@ class Builder::Impl {
 
     unique_ptr<Node> case_(const token *case_, unique_ptr<Node> expr, sorbet::parser::NodeVec whenBodies,
                            const token *elseTok, unique_ptr<Node> elseBody, const token *end) {
+        auto loc = tokLoc(case_);
+        if (end != nullptr) {
+            loc = loc.join(tokLoc(end));
+        }
         return make_unique<Case>(tokLoc(case_).join(tokLoc(end)), std::move(expr), std::move(whenBodies),
```

Comment:
Surely this `loc` addition would be better if it was used in the `make_unique` call?  Or was this an experiment that turned out to be unnecessary?

(Slightly concerned that tests didn't seem to exercise the `end == nullptr` case, too, but maybe that is just a consequence of this being dead code.)

Reply:
Yeah, I started trying to reuse the `case_` builder for the error handling cases, and then at some point realized that it wouldn't be needed. This code is dead.